### ISSUE TITLE
Feature suggestion: add an option to output to a windowed screen instead of fullscreen

### ISSUE THREAD

**Issue by macdelacruz** [vassbo/freeshow#153]
How about a useful feature that will allow users to use OBS and capture a window output of Freeshow. My set up in OBS, I use the custom browser dock feature to load the Freeshow remote. With a window output instead of fullscreen, I can place the window beneath OBS and hence I will not need a third display.

**Comment by vassbo** [comment]
Good suggestion! I'll implement this.
In the meantime you can check the "AlwaysOnTop" option in the "Advanced settings" in the output settings, and you will be able to put other windows on top.

**Comment by macdelacruz** [comment]
Using 0.9.0 now on Windows and I see _Always on top_ and _Kiosk mode_ as new options in the Output settings. Not sure what _Kiosk mode_ does. But yes, turning off _Always on top_ allows putting other windows on top, but still OBS can't capture the output of Freeshow using Window Capture in OBS.

So what I am suggesting maybe you can replicate what OBS does. It outputs in a separate window as you can see when you hover on it on the taskbar. I think if Freeshow can do the same, OBS Window Capture will be able to capture the Freeshow output window.

![OBS-windowed-output](https://github.com/vassbo/freeshow/assets/13848476/ded46993-1142-4702-a8a7-a6f9b5adf427)

**Comment by vassbo** [comment]
Right, I'll probably add an option to show the window in the taskbar.

**Comment by macdelacruz** [comment]
Cool. It works like magic in OBS now. I can capture the Freeshow window now and tada, I can stream, record, apply filters like opacity, add any overlay, the possibilities are endless.

![screen-capture](https://github.com/vassbo/freeshow/assets/13848476/7e64184a-a92f-4b8a-86ee-73c773f304b3)

**Comment by vassbo** [comment]
Cool! I can make it so it always displays when not in fullscreen.
You can hold CTRL while hovering over the output window to show a top bar to drag it from!
